source/_integrations/omnilogic.markdown

```diff
@@ -22,12 +25,44 @@ There is currently support for the following device types within Home Assistant:
 
 - ***Sensor*** - Air Temperature, Water Temperature, Variable Pump Speed, Chlorinator Setting, Salt Level, pH, and ORP
 - ***Switch*** - All relays, pumps (single, dual, variable speed), and relay-based lights.
+- ***Light*** - Colorlogic Lights (V1 and V2).
 
 {% include integrations/config_flow.md %}
 
+## Sensor Platform Options
+
+If you have pH sensors in your Omnilogic setup, you can add an offset to correct reporting from the sensor in the integration configuration.
+
+## Switch Platform
+
+The switch platform contains a custom service to allow you to set the speed on variable speed pumps.
+
+To set pump speed, call the `omnilogic.set_pump_speed` service as following:
+
+```yaml
+service: omnilogic.set_pump_speed
+data:
+  entity_id: Entity ID of the Pump
+  speed: Speed (0-100%)
+```
+
+## Light Platform
+
+The light platform allows you to set the color or effect of your lights from the effect list supported by your light version.
+
+If you have V2 Colorlogic lights you can also set the brightness and speed of the lights using the custom service `omnilogic.set_v2_lights` as following:
+
+```yaml
+service: omnilogic.set_v2_lights
+data:
+  entity_id: Entity ID of the Lights
+  speed: Speed (0-8) for the light effect (optional)
+  brightness: Brightness (0-4) for the lights (optional)
```

Comment:
Brightness should be part of the light entity right? Why control these via this service?

Reply:
We did this as a service because the brightness has only 4 brightness levels. It isn't a "dimmable" light (0-100) but rather more like a fan (low, medium-low, medium, high). I tried implementing with the brightness on the lights, but artificially creating 4 light levels based on the % setting wasn't very user friendly.